grails-app/services/org/pih/warehouse/inventory/ProductAvailabilityService.groovy

```diff
@@ -492,8 +492,8 @@ class ProductAvailabilityService {
         if (products.size() != quantityMap.size()) {
             def missingProducts = products - quantityMap.keySet()
```

Comment:
This will not work, because products is a list of products and keySet is a list of ids, also I'm not sure it's needed, you can just go through the entire list (like it's doing right now)

Reply:
You right, I missed that.

Reply:
@pmuchowski fixed